### ISSUE TITLE
bibm ads-update

### ISSUE THREAD

**Issue by AaronDavidSchneider** [pcubillos/bibmanager#72]
Hi,

I have a small bug to report. Sometimes, when I hit `bibm ads-update` and the shell returns
```bash
There were 1 entries updated from ArXiv to their peer-reviewed version.
```

I get the following error upon compiling a bibliography (`bibm pdflatex`):
```TeX
! Undefined control sequence.
l.466   [\eprint
                [arXiv]{2009.09444}]
```

here is the corrisponding bibliography entry:
```TeX
pdf: Notsu2020_arXiv_2009_9444.pdf
@ARTICLE{Notsu2020,
       author = {{Notsu}, Shota and {Eistrup}, Christian and {Walsh}, Catherine and
         {Nomura}, Hideko},
        title = "{The composition of hot Jupiter atmospheres assembled within chemically evolved protoplanetary discs}",
      journal = {\mnras},
     keywords = {astrochemistry, protoplanetary discs, ISM: molecules, planets and satellites: gaseous planets, planets and satellites: atmospheres, Astrophysics - Earth and Planetary Astrophysics, Astrophysics - Solar and Stellar Astrophysics},
         year = 2020,
        month = sep,
          doi = {10.1093/mnras/staa2944},
archivePrefix = {arXiv},
       eprint = {2009.09444},
 primaryClass = {astro-ph.EP},
       adsurl = {https://ui.adsabs.harvard.edu/abs/2020MNRAS.tmp.2754N},
      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}
```
I can silene the error by removing the eprint line.

Tested:
removed entry completely using `bibm edit` and readded with
```bash
bibm ads-add 2020MNRAS.tmp.2754N Notsu2020
```
The error is still the same. Looks like this is an ADS error?

**Comment by pcubillos** [comment]
Hi, thanks for the report, I will take a look, but my gut feeling is that this is a .bst issue.

**Comment by AaronDavidSchneider** [comment]
ok thanks!

**Comment by pcubillos** [comment]
Hi Aaron,
Sorry for the late answer, I was having too much fun with the other issue (btw, I noticed you looked at the browser issue, feel free to try the draft I wrote and give me your feedback if you want).

So, yeah, I can't reproduce your error when I include that entry into my bibtex file.  But this is what I think is going on:
- Your .bst file is pulling the eprint value from the bibtex entries and creating an \eprint{...} command into the bibliography (this is the bibtex command creating the .bbl file).  (this is not the problem)
- when you compile the latex file (the latex command after you ran bibtex), either your .tex file or the .cls file need to have defined the \eprint{} command (sometimes the .bst file defines the \eprint command as well).

- So, I'm guessing that you have incompatible files where the .bst is making the \eprint{} calls, but latex does not know how to interpret it.  The solution would be to define it in your latex file.  Try adding this line into the header of your .tex file:
```latex
\newcommand{\eprint}[1]{\href{http://arxiv.org/abs/#1}{#1}}
```
If that doesn't work, try with this:
```latex
\renewcommand{\eprint}[1]{\href{http://arxiv.org/abs/#1}{#1}}
```

Let me know if that works!
